framework/org.eclipse.concierge/src/org/eclipse/concierge/Concierge.java

```diff
@@ -4658,7 +4658,11 @@ public Bundle installBundle(final String location, final InputStream in)
 
 			// TODO: check AdminPermission(new bundle, LIFECYCLE)
 
-			return installNewBundle(this, location, in);
+			if(in == null){

```

Comment:
We should agree on a common code format and check-in the .settings/* into project(s).
AFAIK there is a way how code-formatting can be applied during commit. I will check how to enable that